config/nav.yml

```diff
@@ -103,6 +103,7 @@ nav:
           - Configuring certificate class: serving/services/certificate-class.md
           - Configuring custom domains: serving/services/custom-domains.md
           - Using a custom TLS certificate for DomainMapping: serving/services/byo-certificate.md
+          - How to use extensions enabled by QPoptions: serving/services/using-queue-extensions.md
```

Comment:
```suggestion
          - Using extensions enabled by QPoptions: serving/services/using-queue-extensions.md
```

Reply:
Make the name similar to the other items

Reply:
I'm wondering whether it would be better to say "Queue Proxy options" instead of "QPoptions"? 

Also, I've seen "QPOptions" and "QPoptions". Make sure one or the other is consistently used.

Reply:
I will align all to QPOptions

Reply:
replacing to Using extensions enabled by QPoptions need to be aligned with links  - in a separate commit